### PR TITLE
Increase information density in web apps

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
https://dimagi-dev.atlassian.net/browse/USH-3868

- Removes question type text
- Adjusts control height for select2s so they match text inputs using `.form-control`
- Darkens font color in text inputs
- Reduces label font size to match text input font size

before
Screen Shot 2023-11-13 at 8 24 10 PM

after
Screen Shot 2023-11-13 at 8 26 05 PM

### Safety story
Visible but low-risk changes.

### Automated test coverage

no

### QA Plan
Not planning on requesting formal QA. Will be working with delivery to test on staging. Marked "don't merge" until that's done.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/select2s.less [hunk 1]
```diff
@@ -57,13 +57,25 @@
...
     .select2-search-field {
         width: 100% !important;
     }
 
     .select2-input {
         width: 100% !important;
     }
+
+    textarea.select2-search__field {    // Placeholder for multi-select
+        line-height: 21px;
```
Comment (nospame) [inline corehq/apps/hqwebapp/static/hqwebapp/less/_hq/select2s.less:77]: Where does 21px come from (or what does it correspond to)?

**Review by nospame — APPROVED** [review]
Looks good, one non-blocking question.

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/select2s.less (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/select2s.less:77]: It'd be good for it not to be hard-coded, but it doesn't really relate to anything else. It's just from me futzing with styles by hand to center the text vertically. There are alternatives - change the height of the textarea, or its margin - but they would be similar ad hoc values.

**Review by Jtang-1 — APPROVED** [review]

**Comment by orangejenny** [comment]
Re-requesting review. I committed the sin of rebasing and force pushing, because this PR had grown to be 68.75% commits that were in some way reverting or re-reverting previous commits, or re-generating bootstrap 5 diffs, and it had conflicts with B5 diffs on master.

As was noted in some of the revert and re-revert commit messages, this PR does not need to change any bootstrap 5 styling files, because B5 fixes the input height inconsistency issue and also uses the same color for both the default font and the text inside text inputs. Thank you @biyeun for helping to resolve this.

The PR is now just the same set of commits it was when I created it, plus the output of `build_bootstrap5_diffs`, but with all the other B5 faffing about removed.

**Review by nospame — APPROVED** [review]

**Pull request merged**